### PR TITLE
Add new DALP links to homebuyer header [DAH-668]

### PR DESCRIPTION
**Pull request by software-project**
DON'T MERGE till 1/25
Resolves [DAH-668](https://sfgovdt.jira.com/browse/DAH-668)

### CODE REVIEW

**Comment by akegan** [comment]
@software-project can we add an env var flag of ADVERTISE_DALP or something so that we don't have to time a deploy for this?

File: app/assets/json/translations/locale-en.json [hunk 1]
```diff
@@ -1040,8 +1040,12 @@
             "get_help": {
                 "other_groups_may_qualify": "Others, like San Francisco teachers and first responders, may qualify for loans.",
                 "some_people_may_qualify": "Some people may qualify for downpayment assistance.",
-                "title": "Get help",
-                "view_full_list_of_programs": "View the full list of programs"
+                "title": "Help with downpayments",
+                "DALP_program_2021": "The 2021 Downpayment Assistance Loan Program (DALP) will begin accepting applications on February 26, 2021.",
```
Comment (akegan) [inline app/assets/json/translations/locale-en.json:1044]: I think we should have all of the translations strings be lower-case. Can you also run grunt translations and see how it handles having a number in the translation key?

**Comment by software-project** [comment]
@akegan ok, sure, will modify it to use an env flag.

**Comment by software-project** [comment]
@akegan I've added feature flags and changed translation keys, so should be good now.

**Comment by akegan** [comment]
@software-project I'm taking a look now, heads up that jasmine tests are failing on circle CI

File: app/assets/json/translations/locale-en.json [hunk 2]
```diff
@@ -1040,8 +1040,14 @@
             "get_help": {
                 "other_groups_may_qualify": "Others, like San Francisco teachers and first responders, may qualify for loans.",
                 "some_people_may_qualify": "Some people may qualify for downpayment assistance.",
-                "title": "Get help",
-                "view_full_list_of_programs": "View the full list of programs"
+                "title": "Help",
```
Comment (akegan) [inline app/assets/json/translations/locale-en.json:1048]: looks like this should still be "Get help"
```suggestion
                "title": "Get help",
```

File: app/assets/json/translations/locale-en.json [hunk 3]
```diff
@@ -1040,8 +1040,14 @@
...
+                "view_full_list_of_programs": "View the full list of programs",
+                "help_with_downpayment": "Help with downpayments",
+                "dalp_program_info": "The 2021 Downpayment Assistance Loan Program (DALP) will begin accepting applications on February 26, 2021.",
+                "learn_more_and_how_to_apply": "Learn more about DALP and how to apply on SF.gov",
+                "more_resources": "More resources",
+                "mohcd_programs": "The Mayor's Office of Housing and Community Development (MOHCD) offers a number of programs to help homebuyers in San Francisco.",
+                "visit_mohcd_programs": "Visit MOHCD Affordable Homebuyer Programs page to learn more."
```
Comment (akegan) [inline app/assets/json/translations/locale-en.json:1050]:
```suggestion
                "visit_mohcd_programs": "Visit the MOHCD Affordable Homebuyer Programs page to learn more."
```

**Review by akegan — COMMENTED** [review]
A couple of minor copy fixes. I also ran grunt translations to alphabetize the translation keys.

I have a question out on the the Jira ticket to see if we should make the bit about Feb 26th customizable otherwise this would be good to go w/ these minor fixes + fixed tests.

**Review by akegan — APPROVED** [review]
Changes look good. Thanks for the updates! I'll push the translations to phrase now


w/ Advertise DALP = true and nothing for DALP_PROGRAM_INFO
image

with "Something something info" as DALP_PROGRAM_INFO
image

w/ ADVERTISE_DALP=false or with the env var missing
image

**Pull request merged**